src/ir/attribute_test.cc

```diff
@@ -0,0 +1,51 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+#include "src/ir/attribute.h"
+
+#include "src/common/testing/gtest.h"
+
+namespace raksha::ir {
+namespace {
+
+class IntAttributeTest
+    : public testing::TestWithParam<std::pair<int, const char*>> {};
```

Comment:
Why `const char *` here? Isn't `absl::string_view` preferred in these cases?

Reply:
Will change it to absl::string_view.